pom.xml

```diff
@@ -24,6 +25,7 @@
 
```

Comment:
could you please add your name to the contributors list ?


<developer>
  <name>Mathieu Carbou</name>
  <email>mathieu.carbou@softwareag.com</email>
  <organization>Terracotta Inc., a wholly-owned subsidiary of Software AG USA, Inc.</organization>
  <organizationUrl>https://terracotta.org</organizationUrl>
</developer>